### PR TITLE
docs: update all-langs.js link

### PR DESCRIPTION
**Pull request by zairahira**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: docs/how-to-work-on-localized-client-webapp.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ You can test the client app in any language available in the [list of languages
 
 If you are testing a new language, create a folder with the language name as the title next to the other languages and copy the JSON files from another language into your new folder.
 
-Add the language to the `client` array as seen above in the [`config/i18n/all-langs.js`](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/config/i18n/all-langs.js) file.
+Add the language to the `client` array as seen above in the [`config/i18n/all-langs.js`](https://github.com/freeCodeCamp/freeCodeCamp/blob/6b4a6a02568b809fc216ea8566ff5df446d1da4e/config/i18n/all-langs.js#L5) file.
```
Comment (gikf) [inline docs/how-to-work-on-localized-client-webapp.md:96]:
```suggestion
Add the language to the `client` array as seen above in the [`config/i18n/all-langs.ts`](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/config/i18n/all-langs.ts) file.
```
That link appears to also be slightly outdated. The same case is for link on line `85`.

Comment (raisedadead) [reply docs/how-to-work-on-localized-client-webapp.md:96]: I agree it's a terrible idea to use line numbers in docs because they will change all the time.

Comment (gikf) [reply docs/how-to-work-on-localized-client-webapp.md:96]: I'd say so, as the file was migrated to ts, no reason to link to the older js version.

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Review by Sboonny — APPROVED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

**Pull request merged**